src/shared/logger/activation.ts

```diff
@@ -101,6 +101,7 @@ export async function activate(
  * @param opts.logPaths Array of paths to output log entries to
  * @param opts.outputChannels Array of output channels to log entries to
  * @param opts.useDebugConsole If true, outputs log entries to currently-active debug console. As per VS Code API, cannot specify a debug console in particular.
```

Comment:
Some things are over-explained and that makes them more confusing. Yet other things have no comments at all...

```suggestion
 * @param opts.useDebugConsole If true, outputs log entries to `vscode.debug.activeDebugConsole`.
```